contractcourt/channel_arbitrator.go

```diff
@@ -837,11 +865,29 @@ func (c *ChannelArbitrator) stateStep(
 	// to be confirmed.
 	case StateCommitmentBroadcasted:
 		switch trigger {
-		// We are waiting for a commitment to be confirmed, so any
-		// other trigger will be ignored.
+
+		// We are waiting for a commitment to be confirmed.
 		case chainTrigger, userTrigger:
 			log.Infof("ChannelArbitrator(%v): noop trigger %v",
 				c.cfg.ChanPoint, trigger)
```

Comment:
this log is now wrong

Reply:
added commit cnct: simplify channel arbitrator state logging